### ISSUE TITLE
The render delay is too obvious

### ISSUE THREAD

**Issue by light0x00** [qjebbs/vscode-plantuml#556]
Every time i change the plantuml file,  the diagram in the preview tab will delay for a long time.
Is there a way can optimize it? 

Thks!

**Comment by raffian** [comment]
Disable  auto preview/update:
```
"plantuml.previewAutoUpdate": false,
```

**Comment by fromSmolsoft** [comment]
@light0x00  Have you figured out how to make rendering of Plant UML faster ?  

It takes approx 2-3 sec to render single component in VSCode but e.g. in JetBrains IDE it's instantaneous regardless of diagram size.  Both are setup to work "locally" AFAIK, so i can work on the go without connection. 

@raffian `"plantuml.previewAutoUpdate": false,` Doesn't seem to make rendering any faster. Instead it seems to turn off rendering on change.

**Comment by yam-liu** [comment]
yes, extremely slow respond, can I set it to instant render while typing (any text changes)?
